ReadMe.md

```diff
@@ -151,6 +152,7 @@ If you like or are using this project to learn or start your solution, please gi
     - [HangFire](https://github.com/HangfireIO/Hangfire)
     - [Coravel](https://github.com/jamesmh/coravel)
     - [Fluent Scheduler](https://github.com/fluentscheduler/FluentScheduler)
+    - [Quartz.NET](https://www.quartz-scheduler.net/)
```

Comment:
As other developers experience and myself too, Quartz is not really a good task scheduler and it's too complicated. That's why we don't put that in the roadmap.

Reply:
Ok !
I will remove Quartz.NET

Reply:
I removed it.